### ISSUE TITLE
handling outputs in a bosh interpolation

### ISSUE THREAD

**Issue by jtarchie** [pivotal/paving#19]
tl;dr: Can't index JSON arrays in YAML interpolation, making PRs to make it work.

The outputs from terraform can be array values, for example subnet ids from AWS.
This is the correct thing to do, as this is structure data.

At the moment, when trying use those values in a `bosh interpolate`, it is hard to access specific array indexed values.
I'd like to fix this in `bosh int`, and therefore, other CLI utilities that use this.
We found it would be useful for people in the field and Platform Automation specific tasking and configuration.

I'm tracking the work here, too, so people know that this is the original motivation.

**Comment by genevieve** [comment]
@jtarchie It appears a new release of the bosh-cli was cut with your PR: https://github.com/cloudfoundry/bosh-cli/releases/tag/v6.2.0

We can call this out as a dependency in the readme.

**Comment by genevieve** [comment]
Safe to close this issue now?
